lib/hooks/queries/usePortfolioPositions.ts

```diff
@@ -692,10 +745,10 @@ export const usePortfolioPositions = (
   ]);
 
   return {
-    all: positions,
-    markets: marketPositions,
-    subsidy: subsidyPositions,
-    breakdown,
+    all: positions ?? undefined,
+    markets: marketPositions ?? undefined,
+    subsidy: subsidyPositions ?? undefined,
```

Comment:
Seems a bit strange to do this when around line 618 the code does `?? null` why not just do `?? undefined` in the first place?